RocketChat/Rocket.Chat
# [FIX] New users aren't mapped to rooms with OAuth groups/channels map

## Proposed changes (including videos or screenshots)



  This change deals with the problem with the new users coming from key cloak, this users were not being mapped to the correct channels on RC.



## Issue(s)


## Steps to test or reproduce
- Setup a KeyCloak VM and configure it on Rocketchat 
(https://docs.google.com/document/d/1cBohJ1XrLm6vFIdBkEo8sr7xI2huEBE-sCCxcDibZU0/edit)

- enable the setting to map users to channels and login on RC using keycloak, you should be redirected to a channel.


## Further comments


TC-69
